v0/keen/transform.js

```diff
@@ -85,7 +85,11 @@ function processTrack(message, destination) {
   properties.anonymousId = message.anonymousId;
 
   // add ip from the message
-  properties.request_ip = message.request_ip;
+  properties.request_ip = message.context
```

Comment:
@sayan-rudder can you also use the `fixIP` method here

Reply:
done